cart.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Shopping Cart</title>
+</head>
+<body>
+    <header>
+        <a id="logo" href="index.html">
+            <img src="img/bus.png" alt="home" style="width:200px;height:113px;border:0;">
+        </a>
+        <h1>Order Form</h1>
+        <nav>
+            <a href="cart.html">Shopping Cart</a>
+            <a href="bus_mall/index.html">Take a Survey</a>
+        </nav>
+    </header>
+    <main id= "main"></main>
+    <footer></footer>
+    <script src= "app.js"></script>
```

Comment:
Probably do not need to import `app.js` on `cart.html` since you are not using it.